src/core/common/message.hpp

```diff
@@ -443,23 +443,28 @@ class Message : public Buffer
     int CopyTo(uint16_t aSourceOffset, uint16_t aDestinationOffset, uint16_t aLength, Message &aMessage) const;
 
     /**
-     * This method creates a copy of the current Message. It allocates the new one
-     * from the same Message Poll as the original Message and copies @p aLength octets of a payload.
+     * This method creates a copy of the message.
      *
-     * The `Type`, `SubType`, `LinkSecurity` and `Priority` fields on the cloned message are also
+     * It allocates the new message from the same message pool as the original one and copies @p aLength octets
+     * of the payload. The `Type`, `SubType`, `LinkSecurity`, and `Priority` fields on the cloned message are also
```

Comment:
Should add `Offset` and `InterfaceId` to the list of cloned fields.

Reply:
Somehow I totally missed this PR. Just noticed it today. Update the comment.